Makefile

```diff
@@ -82,10 +83,12 @@ rust-components: rust-targets
 rust-tests: rust-targets
 	cargo test
 
-GOLANGCI_LINT_VERSION = "v2.1.6"
```

Comment:
If you remove this, please remove also `linter-version` target which just prints this variable (and now fails).